### DISCUSSION TITLE
Welcome to GAMES-105 Discussions!

### DISCUSSION THREAD

**Discussion by cclib16** [GAMES-105/GAMES-105#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by travmygit** [comment]
Cheers! I've worked in game development for over 3 years and I'm very interested in character animation. I think i will learn a lot in GAMES-105!
